README.md

```diff
@@ -1,3 +1,37 @@
-# Jellyfin Chromecast
+<h1 align="center">Jellyfin Chromecast Client</h1>
+<h3 align="center">Part of the <a href="https://jellyfin.org">Jellyfin Project</a></h3>
 
-Source code for the Jellyfin Chromecast receiver application
+<p align="center">
+<img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
+<br/>
+<br/>
+<a href="https://github.com/jellyfin/jellyfin">
+<img alt="GPL 2.0 License" src="https://img.shields.io/github/license/jellyfin/jellyfin-chromecast.svg"/>
+</a>
+<a href="https://github.com/jellyfin/jellyfin/releases">
+<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-chromecast.svg"/>
+</a>
+</p>
+
+Jellyfin chromecast is a receiver used when casting to a google cast capable device. It is automatically used when casting from the jellyfin android client or jellyfin web client.
+
+### What does it do?
+
+As soon as you push the "cast" button on your client this application will start on you cast-capable device & handle it from there.
```

Comment:
It would probably be good to provide this link for reference to the overall casting process/architecture: https://developers.google.com/cast/docs/overview